diff --git a/Thirdparty/g2o/g2o/core/hyper_graph.h b/Thirdparty/g2o/g2o/core/hyper_graph.h
index bbd5e16..949b501 100644
--- a/Thirdparty/g2o/g2o/core/hyper_graph.h
+++ b/Thirdparty/g2o/g2o/core/hyper_graph.h
@@ -35,11 +35,7 @@
 #include <limits>
 #include <cstddef>
 
-#if defined _MSC_VER || defined __APPLE__
 #include <unordered_map>
-#else
-#include <tr1/unordered_map>
-#endif
 
 
 /** @addtogroup graph */
